src/Illuminate/Auth/Access/Gate.php

```diff
@@ -162,7 +164,7 @@ public function check($ability, $arguments = [])
      *
      * @param  string  $ability
      * @param  array  $arguments
-     * @return callable
+     * @return mixed
```

Comment:
As I see it, by adding `|array` (and in another part of the file `|string`) we might as well change `callable` to `Closure`. Isn't the whole purpose of `callable` to include all those three things?